core/src/main/java/com/datastax/oss/driver/internal/core/os/GraalGetpid.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.core.os;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.graalvm.nativeimage.c.CContext;
+import org.graalvm.nativeimage.c.function.CFunction;
+
+@CContext(GraalGetpid.Directives.class)
+public class GraalGetpid {
+
+  static class Directives implements CContext.Directives {
+
+    @Override
+    public List<String> getHeaderFiles() {
+
+      return Collections.unmodifiableList(Collections.singletonList("<unistd.h>"));
+    }
+
+    /* TODO: I don't know that these are required... */
```

Comment:
After pondering this for a few minutes it became pretty clear that they shouldn't be... it's not like you have to specify these when writing apps in C and it might be weird in the Darwin case.  Regardless things seem to work well without them so I'm removing them.